code/script.js

```diff
@@ -1,19 +1,127 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+alert(`Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`);
+
+let userName = prompt('Please type your name.');
+
+alert(`Welcome ${userName}!`);
 
 // Step 2 - Food choice
-// Your code goes here
+let foodCategory = prompt(`What type of food would you like to order?
+Please enter a number:
+
+1. Pizza
+2. Salad
+3. Kebab`);
 
 // Step 3 - Subtype choice
-// Your code goes here
+let chosenFoodName = '';
+
+switch (foodCategory) {
+    case '1':
+        alert(`You have chosen Pizza`);
+
+        let pizzaChoice = prompt(`What kind of pizza would you like?
+Please enter a number:
+1. Kebab Pizza
+2. Hawaii Pizza
+3. Calzone`);
+
+        switch (pizzaChoice) {
+            case '1':
+                chosenFoodName = 'Kebab Pizza';
+                break;
+            case '2':
+                chosenFoodName = 'Hawaii Pizza';
+                break;
+            case '3':
+                chosenFoodName = 'Calzone';
+                break;
+            default:
+                alert(`Invalid pizza choice`);
+        }
+        break;
+    case '2':
+        alert(`You have chosen Salad`);
+
+        let saladChoice = prompt(`What kind of salad would you like
+Please enter a number:
+1. Ceasar Salad
+2. Avocado Salad
+3. Tuna Salad?`);
+
+        switch (saladChoice) {
+            case '1':
+                chosenFoodName = 'Ceasar Salad';
+                break;
+            case '2':
+                chosenFoodName = 'Avocado Salad';
+                break;
+            case '3':
+                chosenFoodName = 'Tuna Salad';
+                break;
+            default:
+                alert(`Invalid salad choice`);
+        }
+        break;
+    case '3':
+        alert(`You have chosen Kebab`);
+
+        let kebabChoice = prompt(`What kind of kebab would you like?
+Please enter a number:
+1. Meat kebab
+2. Chicken kebab
+3. Vegetarian kebab`);
+
+        switch (kebabChoice) {
+            case '1':
+                chosenFoodName = 'Meat Kebab';
+                break;
+            case '2':
+                chosenFoodName = 'Chicken Kebab';
+                break;
+            case '3':
+                chosenFoodName = 'Vegetarian Kebab';
+                break;
+            default:
+                alert(`Invalid kebab choice`);
+        }
+        break;
+}
+
```

Comment:
Hej Amanda!
1. The code is written very nicely with the nested Switch Case.  But there is a default part that is not written. So when I tested the code I entered an invalid number and the program came up with "You've chosen EMPTY! "
2. I also suggest this part can be used in an Array to store the data. The code will be more efficient.